apps/mobile/src/utils/imagePickers.ts

```diff
@@ -46,7 +45,7 @@ export function moveImageToInternalDirectory({
       if (!rootDirectory) throw new Error('document dir not found')
 
       const parentDirectory = directory
-        ? urlJoin(rootDirectory, directory)
+        ? `${rootDirectory}${directory}`
```

Comment:
Really? This seems weird. I would rather use function for joining urls that concating it directly like this